app/views/form/_date_question.html.erb

```diff
@@ -3,7 +3,7 @@
 <%= f.govuk_date_field question.id.to_sym,
   caption: caption(caption_text, page_header, conditional),
   legend: legend(question, page_header, conditional),
-  hint: { text: question.hint_text&.html_safe || "For example, 1 9 2022." },
+  hint: { text: question.hint_text&.html_safe || "For example, #{Date.today.to_formatted_s(:govuk_date_number_month)}" },
```

Comment:
Do I understand correctly that the form rolls over each year automatically, so today's date should always be in the "current" tax year? Or is there a period in early April where today is not in the tax year of the form currently being filled out?
If the latter, what is it that defines the tax year of the form currently being worked on? If it's the sale completion date we will have a circular dependency here!

Reply:
There's a period of overlap in the collection windows. The window for entering logs for a particular tax year is about 15 months, from 1st April to June/July of the next year. Which version of the form is used depends on which tax year the tenancy start date is in. For instance, if I'm filling out a log on 10th April 2023, but my tenancy start date is 10th *March* 2023, I should be shown the previous version of the form.
A lot of organisations (against DLUHC's advice) wait until the end of the year to submit all of their data, so there is a lot of data that is not submitted for the current tax year.